api/models.py

```diff
@@ -178,7 +178,7 @@ class ImagesWinery(models.Model):
 
 
 class ImagesEvent(models.Model):
-    filefield = models.FileField()
+    filefield = models.FileField(blank=False, null=False, default='')
```

Comment:
Why is the "default=' ' " needed?

Reply:
It's related with a security check to avoid inconsistencies in the database.
If you had previously a model that accept null and rows with null value exist in these specific column, when you perform a migration theses null values are changed to the default value to avoid inconsistencies with the new defined model.

Reply:
Django will assume blank=False and null=False if you don't specify their values, so it's redundant to set them this way.